### ISSUE TITLE
Type group ZAGUT cannot be created

### ISSUE THREAD

**Issue by mrsimpson** [larshp/abapGit-Unit-Test#1]
Test dangerous fails - at least on my system

![2016-03-02 19_17_13-starten](https://cloud.githubusercontent.com/assets/17176678/13470210/7850dfc0-e0ab-11e5-8ff9-b2a8a5402e9e.png)

**Comment by larshp** [comment]
hmm, what SAP version?

I tried on a CAL 7.50 instance a few days ago and it seemed to work, however something might have changed.

**Comment by mrsimpson** [comment]
SAP_BASIS	740	0008

**Comment by larshp** [comment]
thanks,

can you help trying:
a: creating the type group manually via SE11, to rule out authorization issues
b: try installing https://github.com/larshp/abapOpenChecks it also has a type group but it is named ZZAOC this might be

**Comment by mrsimpson** [comment]
Will do that once I've got some time

**Comment by mrsimpson** [comment]
> ZAGUT is already defined as structure or table
> Message no. D0751
> 
> Diagnosis
> You tried to create an object with a name which was already assigned to another object of another type.
> 
> 
> Procedure
> Choose another name for your object.

But there's no TABL or DTEL in the system anymore.
Breakpoint at MESSAGE RAISING is always a good idea.

However, the message looks like a false positive to me:
![2016-03-03 10_54_24-einstellungen](https://cloud.githubusercontent.com/assets/17176678/13491038/571bf030-e130-11e5-8943-c9af8d223506.png)

![2016-03-03 11_07_58-starten](https://cloud.githubusercontent.com/assets/17176678/13491039/571bfeb8-e130-11e5-8c25-4a88f83be819.png)

Only workaround would be to use different names for the objects in the UT - or implement a note I don't know. Or wait for 7.50...

**Comment by larshp** [comment]
did it work creating it manually?

perhaps there is a TABL ZAGUT entry in TADIR in a transportable package that has not been deleted yet due to non released transport

it looks like the type group ZAGUT is the only thing named ZAGUT in this repository

**Comment by mrsimpson** [comment]
nope, what you saw above are screenshots from creating it manually.
There are no entries in TADIR for ZAGUT for any type, obj_exists evaluates to false.
Found the source of error though:
![2016-03-03 11_34_12-starten](https://cloud.githubusercontent.com/assets/17176678/13492405/195683a2-e138-11e5-90be-36ee1d009b8b.png)

Funny enough, this is due to a structure I created as part of the BOPF unit-test.
In the class-teardown, I deleted the structure again. That works fine for all the structures - except the one mentioned. Although DDIF_OBJECT_DELETE says it was deleted, it's available after execution of the unittest.

=> Type-pools seem to be very greedy with respect to the namespace they cover (actually, I've just created the first type-pool in my whole SAP career).
Thus, I'd rename the unittest-type-group as the prefix ZAGUT is a primer for other objects.
